experiment_executor.py

```diff
@@ -138,16 +139,15 @@ def experiment_one(self):
 
         for parallel_requests in range(min_parallel_requests, max_parallel_requests + 1, step):
             self.parallel_requests = parallel_requests
+            cloud_load_lvl = processing_time / (periods_in_vector * self.num_of_nodes * parallel_requests)
 
             for algorithm in self.algorithms:
-                cloud_load_lvl = processing_time / (periods_in_vector * self.num_of_nodes * parallel_requests)
-
-                self.shard_allocation(algorithm). \
-                    calculate_imbalance_level(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl). \
-                    simulation(algorithm). \
-                    calculate_delays(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl). \
-                    estimate_delays(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl)
+                self.run_experiment(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl)
 
+        self.experiment_static_params = "Nodes: %d \nShards: %d \nLoad μ: %.2f \nLoad σ: %.2f " % (self.num_of_nodes,
```

Comment:
this could a class method that adds this property

Reply:
Added one class method